vscode/microsoft-kiota/src/utilities/status-bar.ts

```diff
@@ -0,0 +1,25 @@
+import * as vscode from "vscode";
+
+import { ExtensionContext, StatusBarItem } from "vscode";
+import { getKiotaVersion } from "./getKiotaVersion";
+import { kiotaOutputChannel } from "./logging";
+
+async function updateStatusBarItem(context: ExtensionContext, kiotaStatusBarItem: StatusBarItem): Promise<void> {
+  try {
+    const version = await getKiotaVersion(context, kiotaOutputChannel);
+    if (!version) {
+      throw new Error("kiota not found");
+    }
+    kiotaStatusBarItem.text = `$(extensions-info-message) kiota ${version}`;
```

Comment:
Kiota not kiota?

Reply:
That's from a refactor. The strings were carried as they were. I am not sure if changing them here would affect the localization downstream